include/ensmallen_bits/cmaes/cmaes_impl.hpp

```diff
@@ -171,14 +177,14 @@ typename MatType::elem_type CMAES<SelectionPolicyType>::Optimize(
       else
       {
         pStep[idx(j)] = arma::randn<BaseMatType>(iterate.n_rows, iterate.n_cols)
-            * covLower;
+            * covLower.t();
```

Comment:
Can you explain why this change is needed?  It isn't immediately obvious to me.

Reply:
This is the case when the parameters are represented as a row vector. In the case of a column vector, `pStep[idx(j)] = covLower * arma::randn<BaseMatType>(iterate.n_rows, iterate.n_cols)`. The same parameters represented as a row vector would use the trasposed form: `pStep[idx(j)] = arma::randn<BaseMatType>(iterate.n_rows, iterate.n_cols) * covLower.t()`. I've explained this inconsistency in point one of this PR.